subsys/bluetooth/mesh/sensor.c

```diff
@@ -636,45 +478,30 @@ int sensor_cadence_decode(struct net_buf_simple *buf,
 			  struct bt_mesh_sensor_threshold *threshold)
 {
 	const struct bt_mesh_sensor_format *delta_format;
-#ifdef CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE
-	enum bt_mesh_sensor_delta delta_type;
-#else
 	enum cadence_delta_type delta_type;
-#endif
+
 	uint8_t div_and_type;
 	int err;
 
 	div_and_type = net_buf_simple_pull_u8(buf);
 	delta_type = div_and_type >> 7;
-#ifdef CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE
-	threshold->delta.type = delta_type;
-	delta_format = (delta_type == BT_MESH_SENSOR_DELTA_PERCENT) ?
-			&bt_mesh_sensor_format_percentage_delta_trigger :
-			sensor_type->channels[0].format;
-#else
+
```

Comment:
Not needed empty line

Reply:
There are several not-needed empty lines in this file, please remove all, it makes it look like there is some structure that is not actually there